monitoring/uss_qualifier/requirements/astm/f3548/v21.md

```diff
@@ -0,0 +1,167 @@
+# ASTM F3548-21
+
+_Standard Specification for UAS Traffic Management (UTM) UAS Service Supplier (USS) Interoperability_
+
+For information on these requirements, refer to [the ASTM standard F3548-21](https://www.astm.org/f3548-21.html).
+
+## Common Requirements
+
+* <tt>GEN0005</tt>
+* <tt>GEN0010</tt>
+* <tt>GEN0015</tt>
+* <tt>GEN0100</tt>
+* <tt>GEN0105</tt>
+* <tt>GEN0200</tt>
+* <tt>GEN0300</tt>
+* <tt>GEN0305</tt>
+* <tt>GEN0310</tt>
+* <tt>GEN0400</tt>
+* <tt>GEN0405</tt>
+* <tt>GEN0500</tt>
+
+## Operational Intent Creation and Modification
+
+* <tt>OPIN0005</tt>
+* <tt>OPIN0010</tt>
+* <tt>OPIN0015</tt>
+* <tt>OPIN0020</tt>
+* <tt>OPIN0025</tt>
+* <tt>OPIN0030</tt>
+* <tt>OPIN0035</tt>
+* <tt>OPIN0040</tt>
+* <tt>USS0005</tt>
+
+## Strategic Conflict Detection Service
+
+* <tt>SCD0005</tt>
+* <tt>SCD0010</tt>
+* <tt>SCD0015</tt>
+* <tt>SCD0020</tt>
+* <tt>SCD0025</tt>
+* <tt>SCD0030</tt>
+* <tt>SCD0035</tt>
+* <tt>SCD0040</tt>
+* <tt>SCD0045</tt>
+* <tt>SCD0050</tt>
+* <tt>SCD0055</tt>
+* <tt>SCD0060</tt>
+* <tt>SCD0065</tt>
+* <tt>SCD0070</tt>
+* <tt>SCD0075</tt>
+* <tt>SCD0080</tt>
+* <tt>SCD0085</tt>
+* <tt>SCD0090</tt>
+* <tt>SCD0095</tt>
+* <tt>SCD0100</tt>
+* <tt>GEN0500</tt>
+* <tt>USS0105</tt>
+
+## Aggregate Operational Intent Conformance Monitoring Service
+
+* <tt>ACM0005</tt>
+* <tt>ACM0010</tt>
+* <tt>ACM0015</tt>
+
+## Conformance Monitoring for Situational Awareness Service
+
+* <tt>CMSA0005</tt>
+* <tt>CMSA0010</tt>
+* <tt>CMSA0015</tt>
+* <tt>CMSA0020</tt>
+* <tt>CMSA0025</tt>
+* <tt>CMSA0030</tt>
+* <tt>CMSA0035</tt>
+* <tt>CMSA0040</tt>
+* <tt>CMSA0100</tt>
+* <tt>CMSA0105</tt>
+* <tt>CMSA0110</tt>
+* <tt>CMSA0115</tt>
+* <tt>CMSA0200</tt>
+* <tt>CMSA0205</tt>
+* <tt>CMSA0210</tt>
+* <tt>CMSA0215</tt>
+* <tt>CMSA0300</tt>
+* <tt>CMSA0305</tt>
+* <tt>CMSA0310</tt>
+* <tt>CMSA0315</tt>
+* <tt>CMSA0320</tt>
+* <tt>CMSA0325</tt>
+* <tt>CMSA0330</tt>
+* <tt>USS0105</tt>
+* <tt>USS0105</tt>
```

Comment:
For Strategic Conflict Detection Service, you removed duplicated USS0105 (should be required 3 times) whereas here it is mentioned twice. Is there a particular reason?

Reply:
The intent was to not have duplicates -- will remove.